example/dev/RGBD/README.md

```diff
@@ -0,0 +1,137 @@
+# Example of RGBD device usage
+
+
+A user can interact with a YARP device driver directly inside the local application or via client/server paradigm through the network. Here are example for both cases.
+
+## Smoke test
+
+Let's run a quick test to see how YARP devices can run with a few simple commands. First we need to compile YARP repository, setting the following CMake flags to `true` in order to compile all required modules:
+
+```
+cmake .. -DCREATE_GUIS=true -DCREATE_OPTIONAL_CARRIERS=true -DENABLE_yarpcar_portmonitor=true -DENABLE_yarpcar_depthimage=true -DENABLE_yarpmod_fakeDepthCamera=true
+```
+
+To fully run the test, we need to open 5 terminals for command line input.
+
+First we need to run the yarpserver and the device driver producing the images. In this example we will use a fake device, for testing purposes.
+
+```
+Terminal 1: yarpserver
+Terminal 2: yarpdev --device RGBDSensorWrapper --name /server --subdevice fakeDepthCamera --mode ball 
```

Comment:
Command could be shorter if we appended `wrapper RGBDSensorWrapper` to `src/devices/fakeDepthCamera/fakeDepthCamera.ini` (I think touching `src/devices/fakeDepthCamera/CMakeLists.txt` was for static libs). Would also be nicer in general.

Reply:
Yes, you are right, I'll add it.
Anyway in this specific case I put everything on purpose because it is an example, so I prefer to explicitly call every device involved. This helps to understand how and way devices are created.

Reply:
Understood and agreed! Thank you!!